examples/tox21/keras_fcnet.py

```diff
@@ -0,0 +1,60 @@
+"""
+Script that trains multitask models on Tox21 dataset.
+"""
+from __future__ import print_function
+from __future__ import division
+from __future__ import unicode_literals
+
+import numpy as np
+import keras
+from keras import Input, Model
+from keras.layers import Dense, Dropout, Multiply
+import tensorflow as tf
+
+import deepchem as dc
+
```

Comment:
Simple models are really pretty, easy and clean!

Reply:
Yes, agreed! It's also great since anyone who knows TensorFlow can immediately see how to train DeepChem models.